src/simularium/WebsocketClient.ts

```diff
@@ -47,6 +47,9 @@ export const enum NetMessageEnum {
     ID_AVAILABLE_METRICS_RESPONSE = 18,
     ID_PLOT_DATA_REQUEST = 19,
     ID_PLOT_DATA_RESPONSE = 20,
+    ID_ERROR_MSG = 21,
+    ID_CHECK_HEALTH_REQUEST = 22,
+    ID_SERVER_HEALTHY_RESPONSE = 23,
```

Comment:
is there any documentation in this repo for what these messages are supposed to mean, or their payload schema?  (One thing that is nice to know is whether it's intended to go from client to server or vice versa)

Reply:
There isn't... I do have it fairly well documentation in the Octopus repo though, so I should probably copy that over

Reply:
Yeah and that's true of the message types themselves. We're already in a slightly weird situation, may as well have it be a better documented weird situation!